custom_typings/custom.d.ts

```diff
@@ -0,0 +1,6 @@
+declare var PR;
+declare var System;
+
+declare namespace kendo {
```

Comment:
for some reason Kendo's typescript definitons do not expose the jQuery property they all have